packages/node_modules/@webex/plugin-meetings/src/members/request.js

```diff
@@ -63,6 +63,16 @@ export default class MembersRequest extends StatelessWebexPlugin {
 
     return this.request(requestParams);
   }
+  
+  raiseLowerHandMember(options) {
```

Comment:
minor: I think the name of the function is not descriptive enough
`raiseOrLowerHandMember` or `setRaiseHandMemberState`

Reply:
adressed - pls check again